lib/Db/Alias.php

```diff
@@ -17,8 +17,8 @@
 /**
  * @method void setAccountId(int $accountId)
  * @method int getAccountId()
- * @method void setName(string $name)
- * @method string getName()
+ * @method void setName(string|null $name)
+ * @method string|null getName()
```

Comment:
Not strictly related but I noticed that the DB schema allows null values but the entity does not.